.expeditor/config.yml

```diff
@@ -17,7 +22,12 @@ slack:
 merge_actions:
   - built_in:update_changelog:
       ignore_labels:
-        - "Expeditor: Exclude from Changelog"
+        - "Expeditor: Skip Changelog"
+        - "Expeditor: Skip All"
+  - built_in:bump_version:
```

Comment:
You may want to set `github.version_tag_format` to be something like `fake-{{version}}` or something. Otherwise, by default, it will create a tag matching your "fake version"